contrib/node/tests/python/pants_test/contrib/node/targets/test_node_thrift_library.py

```diff
@@ -0,0 +1,18 @@
+from pants_test.test_base import TestBase
+from pants.contrib.node.targets.node_thrift_library import NodeThriftLibrary
+
+
+class NodeThriftLibraryTest(TestBase):
+  def test_bin_executables_string(self) -> None:
```

Comment:
I think it'll be more useful to check the target types here plus any checks of synthetic/codegen